test/e2e/reconfigure_test.go

```diff
@@ -46,10 +46,6 @@ func (tc *testContext) reconfigurationTest(t *testing.T) {
 	_, err = tc.client.K8s.CoreV1().Nodes().Patch(context.TODO(), machineNodes[0].Name, types.JSONPatchType,
 		patchData, metav1.PatchOptions{})
 	require.NoError(t, err)
-	// Ensure operator communicates to OLM that upgrade is not safe when processing Machine nodes
-	err = tc.validateUpgradeableCondition(metav1.ConditionFalse)
```

Comment:
In your other PR, you removed both `validateUpgradeableCondition()` calls from the reconfiguration test. Did you only remove one on purpose?

I still don't understand why this is not a bug.

Reply:
I removed only one because @saifshaikh48 still wanted the second one if it was not causing issues, which it was not.

This isn't a bug because its an issue with our end to end tests, the operator is functioning properly.

Reply:
@sebsoto like we were discussing on Slack, you have discovered a scenario where WICD is working but WMCO is not and is marking the operator as upgradable. That is a bug. I am fine if you open a bug, comment out the code, add a comment linking the bug and saying this code needs to be uncommented as part of fixing the bug.